extensions/VisualEditor/ApiVisualEditor.php

```diff
@@ -163,6 +163,16 @@ protected function getHTML( $title, $parserParams ) {
 			$timestamp = $latestRevision->getTimestamp();
 		} else {
 			$content = '';
+			$preloadTitle = Title::newFromText( urldecode( $parserParams['preload'] ) );
```

Comment:
I believe you should not have to urldecode - ApiMain/Base should already have handle it for you.